src/System.Data.Odbc/tests/System.Data.Odbc.Tests.csproj

```diff
@@ -25,6 +25,21 @@
     <Compile Include="TestCommon\CheckConnStrSetupFactAttribute.cs" />
     <Compile Include="OdbcParameterTests.cs" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsLinux)' == 'true' And '$(IsPartialFacadeAssembly)' != 'true' ">
```

Comment:
IsPartialFacadeAssembly will always be false when TargetsLinux == true or TargetsOsx == true